linux/Dockerfile

```
@@ -94,7 +94,7 @@ ARG musl_target
 RUN git clone https://github.com/crystal-lang/shards \
  && cd shards \
  && git checkout ${shards_version} \
- && shards install --production \
+ && make lib \
  \
  && /crystal/bin/crystal build --stats --target ${musl_target} \
     ./src/shards.cr -o shards --static ${release:+--release} \
```

We could also consider to use `make` for building the binary. 
```suggestion
 && CRYSTAL_OPTS="--stats --target ${musl_target} --static ${release:+--release}" make \
```

---

That  reminds me we should give the shards Makefile  a `release=1` option like we got for the crystal one.

---

I want to unify a bit the makefile arguments of shards and crystal a bit. Like in https://github.com/crystal-lang/shards/compare/master...bcardiff:feature/makefile-flags

That will need to wait for a shard release before it is used here.

Mind if we update first that and then we unify how shards is build across the scripts?